File: SE201-main/SE201-PZ-Bolnica-StefanGogic4056/SE201-PZ-Bolnica/src/abstraktneKlase/Osoba.java
package abstraktneKlase;

public abstract class Osoba {

    public String ime;
    public String prezime;

    public String getIme() {
        return ime;
    }

    public void setIme(String newIme) {
        ime = newIme;
    }

    public String getPrezime() {
        return prezime;
    }

    public void setPrezime(String newPrezime) {
        prezime = newPrezime;
    }

    public Osoba() {
    }

    public Osoba(String ime, String prezime) {
        this.ime = ime;
        this.prezime = prezime; 
    }

}

File: CS322-C#/CS322-DZ13/CS322-DZ13/CS322-DZ13/Osoba.cs
using System;
using System.Text;

namespace CS322_DZ13
{
	class Osoba
	{
		public string Ime { get; set; }
		public string Prezime { get; set; }
		public string JMBG { get; set; }

		public Osoba(string Ime, string Prezime, string JMBG)
		{
			this.Ime = Ime;
			this.Prezime = Prezime;
			this.JMBG = JMBG;
		}

	}
}
